CharacterCreation/03_Feats.txt

```diff
@@ -13,76 +13,76 @@ Passive Feats
 ==============================
 Akimbo:
 Prerequisites: Dex 7, Level 6, Dual Wield feat
-Reduce the miss penalty from dual wielding to +1 miss chance per weapon. Reduce
+	Reduce the miss penalty from dual wielding to +1 miss chance per weapon. Reduce
 the reload penalty to 10%. You may fire at more than one target at once for 
 an additional +2 miss chance penalty.
 
-Acquired: Additional Plating
+Acquired, Additional Plating:
 Prerequisites: Marksman, Gunslinger
-Enables the use of Medium Armour by the listed classes.
+	Enables the use of Medium Armor by the listed classes.

```

Comment:
I can't thank you enough for writing these out. That had been bugging me for a while.

Reply:
word